src/main/scala/com/github/unchama/seichiassist/subsystems/seasonalevents/anniversary/AnniversaryItemData.scala

```diff
@@ -100,6 +101,12 @@ object AnniversaryItemData {
   def isMendingBook(item: ItemStack): Boolean =
     item != null && item.getType == WRITTEN_BOOK && {
       new NBTItem(item).getByte(NBTTagConstants.typeIdTag) == 2
+    } && isOriginalBook(item.getItemMeta)
+
+  private def isOriginalBook(meta: ItemMeta) =
+    meta match {
+      case bookMeta: BookMeta => bookMeta.hasGeneration && bookMeta.getGeneration == Generation.ORIGINAL
+      case _ => false
     }
```

Comment:
`private def` のスコープを狭くできます

```suggestion
  def isMendingBook(item: ItemStack): Boolean = {
    def isOriginal(meta: ItemMeta) =
      meta match {
        case bookMeta: BookMeta => bookMeta.hasGeneration && bookMeta.getGeneration == Generation.ORIGINAL
        case _ => false
      }

    item != null && item.getType == WRITTEN_BOOK && {
      new NBTItem(item).getByte(NBTTagConstants.typeIdTag) == 2
    } && isOriginal(item.getItemMeta)
  }
```

Reply:
ありがとうございます！直しました